protos/core.proto

```diff
@@ -167,6 +167,13 @@ message SQLDataWarehouseOptions {
   string distribution = 1;
 }
 
+message PrestoOptions {
+  string catalog = 1;
+  string schema = 2;
+  repeated string partition_by = 3;
+  // TODO: Add map of configurations to different presto connections.
```

Comment:
for my context: can you clarify what you mean here / how it would work? my presumption was that all that config lives inside Presto itself, such that if i query against some catalog, Presto uses whatever connection config it needs to for that catalog.

Reply:
So in your initial connection you can specify a catalog and a schema if desired, then if you query `select * from table` it'll do the same as `select * from catalog.schema.table`. Currently catalog and schema are required by the client (but they shouldn't be), and the plan is for connections to **not have catalog or schema set in the connection at all**, so that there's no confusion about arbitrary table names.

 Having schema and catalog in the config seems like a nice way to allow users to have cross catalog files (default for if there's no catalog or schema defined), or specify a catalog and schema within the config if they don't want to have to write out the full target name for each table.

Reply:
If just catalog is provided then `select * from schema.table` works too

Reply:
The current `database` and `schema` should work fine for this though, so there's no need to add presto specific proto options for them